README.md

```diff
@@ -40,13 +52,31 @@ Log.Logger = new LoggerConfiguration()
       onBeforeSendArguments => { /*OnBeforeSend: Action<onBeforeSendArguments>*/ })
     .CreateLogger();
 ```
-
 ------
 
-## Step 3 - Configure the JSON configuration file
+### Option 2 - Using the JSON configuration file
+You can initialize Raygun's Serilog Sink inside a Serilog JSON configuration file using the following examples.
+**Minimum setup example:**
 
-When configuring using a JSON configuration file use the following example -
+```json
+{
+  "Serilog": {
+    "Using": [
+      "Serilog.Sinks.Raygun"
+    ],
+    "WriteTo": [
+      {
+        "Name": "Raygun",
+        "Args": {
+          "applicationKey": "paste_your_api_key_here"
+		}
+      }
+    ]
+  }  
+}
+```
 
+**Example setup with optional properties:**
 ```json
 {
   "Serilog": {
```

Comment:
The "Getting Started" instructions end at the end of this code snippet. Following that is a section called "Properties". I'm wondering if that should be at heading level 2 instead of 3 to make it clear that you've reached the end of the setup instructions and now leading into optional stuff.
Also, maybe it should be called something like "Configuration options" instead of "Properties" - something to discuss.
Also, perhaps have a horizontal rule after this code block and before the next section for further clarity and consistency with the use of horizontal rules at the end of each step within the Getting started section.